scribble-doc/scribblings/scribble/core.scrbl

```diff
@@ -1421,7 +1444,32 @@ number for each subsection. If @racket[numberer-step] produces a plain
 string for the rendered number, then it is not added as a prefix to
 subsection numbers. See also @racket[collected-info].
 
-@history[#:added "6.4"]}
+@history[#:added "1.1"]}
+
+
+@defstruct[link-render-style ([mode (or/c 'default number)])]{
+
+Used as a @tech{style property} for a @racket[part] or a specific
+@racket[link-element] to control the way that the link is rendered.
```

Comment:
Replace "the way the link is rendered" with "what parts of the `link-element` are hyperlinked" ?